app/views/datagrid/_dynamic_filter.html.erb

```diff
@@ -0,0 +1,22 @@
+<%# locals: (filter:, options: {}, object:, form:, field:, operation:, value:) -%>
+
+<%= form.dynamic_filter_select(filter.name,
```

Comment:
`dynamic_filter_select` has some UI inside: hiding selection when only one option is available.
It should be exposed for customizations too.

Besides that, I think it is a good idea to call `datagrid_filter_input` instead.
That makes inputs rendering controllable and customizable inside `datagrid_filter_input` if the gem itself needs an update.